vm/src/frame.rs

```diff
@@ -860,6 +860,10 @@ impl Frame {
                 self.push_value(formatted);
                 Ok(None)
             }
+            bytecode::Instruction::PopException {} => {
+                assert!(!vm.pop_exception().is_none());
```

Comment:
This can just be `assert!(vm.pop_exception().is_some())`.

Reply:
Done